src/typed.h

```diff
@@ -67,6 +67,7 @@
 #error Unknown precision type
 #endif
 
+#define is_norm_above_threshold(x, t) ((double) (x) > (double) (t))
```

Comment:
All of the places where this macro is called use only `double`. A type cast is not necessary and you don't really need a macro to do `x > t`.